src/Microsoft.AspNet.Mvc.Razor/Resources.resx

```diff
@@ -133,7 +133,7 @@
     <value>The layout view '{0}' could not be located. The following locations were searched:{1}</value>
   </data>
   <data name="LayoutCannotBeRendered" xml:space="preserve">
-    <value>A layout page cannot be rendered after '{0}' has been invoked.</value>
+    <value>A layout page cannot be rendered at '{0}' after '{1}' has been invoked.</value>
```

Comment:
- updated sentence does not scan.  suggest `Layout page '{1}' cannot be rendered after '{0}' has been invoked.`
- I reversed the parameters to match `SectionAlreadyRendered` but that's a nit